### ISSUE TITLE
REQUEST: Migrate https://github.com/oracle/cluster-api-provider-oci

### ISSUE THREAD

**Issue by joekr** [kubernetes/org#3404]
### New repo, staging repo, or migrate existing

migrate existing with a possible rename to `cluster-api-provider-oracle-cloud`

### Is it a staging repo?

no

### Requested name for new repository

cluster-api-provider-oci

### Which Organization should it reside

kubernetes-sigs

### Who should have admin access?

@shyamradhakrishnan, @arindam-bandyopadhyay, @joekr, @Djelibeybi

### Who should have write access?

@shyamradhakrishnan, @arindam-bandyopadhyay, @joekr, @Djelibeybi

### Who should be listed as approvers in OWNERS?

OWNERS file has been added here https://github.com/oracle/cluster-api-provider-oci/pull/81

### Who should be listed in SECURITY_CONTACTS?

SECURITY_CONTACTS file has been added here https://github.com/oracle/cluster-api-provider-oci/pull/81

### What should the repo description be?

Kubernetes Cluster API Provider for Oracle Cloud Infrastructure 

### What SIG and subproject does this fall under?

cluster-api

### Please provide references to appropriate approval for this new repo

* All contributors must have signed the CNCF Individual CLA or CNCF Corporate CLA - Corporate CLA signed
* Boilerplate text across all files should attribute copyright as follows: "Copyright <Project Authors>" if no CLA was in place prior to donation - we have an open PR to do this once migrated https://github.com/oracle/cluster-api-provider-oci/pull/81
* Additions of [the standard Kubernetes header](https://git.k8s.io/kubernetes/hack/boilerplate/boilerplate.go.txt) to code created by the contributors can occur post-transfer, but should ideally occur shortly thereafter. - we have an open PR to do this once migrated https://github.com/oracle/cluster-api-provider-oci/pull/81
* Should contain template files as per the [kubernetes-template-project](https://github.com/kubernetes/kubernetes-template-project). - PR https://github.com/oracle/cluster-api-provider-oci/pull/81
* sig-cloud-provider Group approved migrating this repo request https://docs.google.com/document/d/1mgZxjDbnSv74Vut1aHtWplG6vsR9zu5sqXvQN8SPgCc

### Additional context for request

- Slack channel created: #cluster-api-oci

**Comment by joekr** [comment]
Howdy @bridgetkromhout  👋  🤠 

I will update the issue to ask to have the repo name changed from cluster-api-provider-oci to cluster-api-provider-oracle-cloud-infrastructure  upon donation to help clear up confusion. This also helps keep our naming consistent as well 😄   https://www.oracle.com/in/cloud/

**Comment by neolit123** [comment]
+1 for the transfer on my side.

**Comment by mrbobbytables** [comment]
@joekr it looks like everyone set for owners/admin/write access are not org members, could you open up requests for org membership? If it'd speed things along, just need 1 person to start (feel free to use me as a sponsor to start the ball rolling)

Also do you have permission to migrate the repo? We have a temporary org (https://github.com/kubernetes-purgatory) that we invite external folk to is a staging ground before migrating into the org. For companies it might be an internal github admin.

/assign

**Comment by joekr** [comment]
I've requested membership thanks 😄  https://github.com/kubernetes/org/issues/3499

I don't have permission to migrate. I'll track down the person we will want to handle that.

**Comment by mrbobbytables** [comment]
@Julia3072 @spavlusieva - can one of you ack on here to confirm adding to the other org?

**Comment by mrbobbytables** [comment]
Just checking in, is this still something to be pursued? we need to transfer the repo to the intermediary org before we can bring it in here.

**Comment by joekr** [comment]
@mrbobbytables yes, we are actively working on this, albeit slowly. In order to donate we have to get a lot of sign off, which we are working on.

**Comment by mrbobbytables** [comment]
No worries! just wanted to double check

**Comment by joekr** [comment]
We are still working on getting approval to migrate.

**Comment by joekr** [comment]
Sorry, I promise we are working on this. 😞

**Comment by ameukam** [comment]
@joekr Hi! Any update about this ? Can we freeze this issue until things are unlocked on your side ?

**Comment by joekr** [comment]
I'm still trying to get approval on my end. Sorry it has taken so long. Feel free to freeze it.

**Comment by mrbobbytables** [comment]
We're going on about a year now - is this still ongoing?

**Comment by joekr** [comment]
.... I know ... I know ....

It is still ongoing internally. If this is causing issues on your all's end lets close this then I can reopen it or create a new one. Sorry for the delay.

**Comment by joekr** [comment]
So this doesn't continue to cause issues I'll close this for now. Then just create a new issue when we are actually ready.
